src/main/java/com/amazon/opendistroforelasticsearch/sql/executor/AsyncRestExecutor.java

```diff
@@ -66,17 +71,17 @@
     public void execute(Client client, Map<String, String> params, QueryAction queryAction, RestChannel channel) throws Exception {
         if (isBlockingAction(queryAction) && isRunningInTransportThread()) {
             if (LOG.isDebugEnabled()) {
-                LOG.debug("Async blocking query action [{}] for executor [{}] in current thread [{}]",
-                    name(executor), name(queryAction), Thread.currentThread().getName());
+                LOG.debug("[{}] Async blocking query action [{}] for executor [{}] in current thread [{}]",
+                    requestId(queryAction), name(executor), name(queryAction), Thread.currentThread().getName());
             }
             async(client, params, queryAction, channel);
         }
         else {
             if (LOG.isDebugEnabled()) {
-                LOG.debug("Continue running query action [{}] for executor [{}] in current thread [{}]",
-                    name(executor), name(queryAction), Thread.currentThread().getName());
+                LOG.debug("[{}] Continue running query action [{}] for executor [{}] in current thread [{}]",
+                    requestId(queryAction), name(executor), name(queryAction), Thread.currentThread().getName());
             }
-            executor.execute(client, params, queryAction, channel);
+            sync(client, params, queryAction, channel);
```

Comment:
the name `sync` is a little misleading, I read it as *synchronize* where I assume you meant *executeSynchronously*.

Reply:
Yeah, probably, I use it to be consistent with the method `async`. So do you think it's better to change both to their full name `executeSynchronously` and `executeAsynchronously`?

Reply:
it's a little too long, but I think it's still better to have more explicit/descriptive naming, so yes.

Reply:
As Jing's comments, will remove this wrapper method. Resolving this conversation.